examples/player/simid_player.html

```diff
@@ -42,6 +42,27 @@
       simidPlayer.playAd();
     }
 
+    function closeAd() {
+      if (!simidPlayer) {
+        initAd();
+      }
+      simidPlayer.closeAd();
+    }
+
+    function skipAd() {
+      if (!simidPlayer) {
+        initAd();
+      }
+      simidPlayer.skipAd();
+    }
+
+    function fatalError() {
+      if (!simidPlayer) {
+        initAd();
```

Comment:
Nope. We shouldn't init an ad here. same comment as above.

perhaps you could code it this way too.
if (simidPlayer) {
  simidPlayer.fatalError();
}

no reason to init an ad in this function.

Reply:
okay that's what I originally did but based on PR comments I changed it. Will revert back